core/citrus-base/src/main/java/org/citrusframework/DefaultTestCase.java

```diff
@@ -51,6 +51,8 @@ public class DefaultTestCase extends AbstractActionContainer implements TestCase
 
     private static final Logger logger = LoggerFactory.getLogger(DefaultTestCase.class);
 
+    private static final TestResultInstanceProvider DEFAULT_TEST_RESULT_INSTANCE_PROVIDER = new DefaultTestResultInstanceProvider();
```

Comment:
am I missing something, or where is this class? have been [searching for it](https://github.com/search?q=repo%3Acitrusframework%2Fcitrus%20DefaultTestResultInstanceProvider&type=code).

Reply:
It is here:
https://github.com/citrusframework/citrus/pull/1182/files#diff-62108b235ff16c88ad3d6c58aeba60b76f84d55e98f54abe596be03401f471c1R547

Reply:
haha - I thought "why is this pipeline passing?!"

damn 🤖